app/src/utils/paraspell.ts

```diff
@@ -73,7 +73,7 @@ export const getTokenSymbol = (sourceChain: TNodeDotKsmWithRelayChains, token: T
   // TODO(victor): write some tests
   const supportedAssets = assets.getAllAssetsSymbols(sourceChain)
   const tokenSymbol = supportedAssets.find(a => a.toLowerCase() === token.symbol.toLowerCase())
-  if (!tokenSymbol) throw new Error('Transfer failed: Token symbol not supported.')
+  if (!tokenSymbol) throw new Error('Token symbol not supported.')
```

Comment:
I think this one is thrown as a notification to the user so we should reduce the technical language here. "Token not supported" is enough and clear for them.

```suggestion
  if (!tokenSymbol) throw new Error('Token not supported.')
```

Reply:
it is not shown to the user. So I keep it